docs/leaderboard.csv

```diff
@@ -1,30 +1,31 @@
+IUT-CSE MLP (keyword),https://github.com/hearbenchmark/hear2021-submitted-models,0.7596691549,0.9109042645,0.4243097842,0.5179487467,0.3674999952,0.1867990792,0.5539999902,0.8891025662,0.9315476247,0.4505244732,0.06481262073,0.9686118007,0.9415217042,0.6046798229,0.4399999976,0.9762781858,0.9779140949,0.05607668435,0.1810890824
+Kuroyanagi hearline,https://github.com/hearbenchmark/hear2021-submitted-models,,0.928102839,0.479967159,,0.4244999945,0.1780913174,0.6540000021,0.9307692349,0.5178571429,0.4979021013,0.001638807221,0.9094165564,0.7834314108,0.5894088745,0.3939999938,0.478323102,0.579754591,0.03982497503,0.1800581545
+Logitech AI SERAB BYOL-S,https://github.com/hearbenchmark/hear2021-submitted-models,0.5486593246,0.9532801509,0.65689677,0.6421319842,0.8049999952,0.5087932348,0.8370000005,0.9384615421,0.8571428571,0.7853146911,0.007860300969,0.9726248145,0.9284779787,0.7115763426,0.3959999979,0.9137014151,0.9480572343,0.1598111937,0.4578059733
+OpenL3,https://github.com/hearbenchmark/hear2021-submitted-models,0.6040219963,0.9745567441,0.5497405887,0.8327796459,0.7505000114,0.4469626248,0.8789999962,0.968589747,0.9494047676,0.6414335728,0.01650246941,0.9666057706,0.9369351149,0.7310344577,0.5600000024,0.6795501113,0.7633946538,0.0781204775,0.3312979102
+RedRice EfficientNet-B2,https://github.com/hearbenchmark/hear2021-submitted-models,0.5331790149,0.9534574509,0.5746188283,0.7900677323,0.9345000029,0.6070509553,0.8779999971,0.9679487228,0.8779761876,0.650874126,0.0001910291583,0.9485457778,0.8431979775,0.3913792968,0.1679999977,0.5734151602,0.6756646037,0.1384509554,0.2551308453
+Soundsensing YAMNet,https://github.com/hearbenchmark/hear2021-submitted-models,0.4664834142,0.940514195,0.4533493698,0.008316008374,0.8375,,0.8469999909,0.968589747,0.7321428571,0.6526223779,0.0,,,0.3209359646,0.1580000073,0.2887525558,0.4104294479,0.0847094506,0.201649484
+Stellenbosch LSL DBERT,https://github.com/hearbenchmark/hear2021-submitted-models,0.6972415149,0.9191489458,0.5215128779,0.2462006211,0.5315000057,0.2634900212,0.6740000069,0.968589747,0.8095238124,0.5842657328,0.0001634800719,0.8351714611,0.6848203301,0.7374384403,0.523999989,0.565644145,0.6304703355,0.1611349632,0.2458524972
+wav2vec2,https://github.com/hearbenchmark/hear2021-submitted-models,,0.9067376018,0.6562244773,0.6629955769,0.5610000014,0.341744781,0.7800000012,0.9461538494,0.8482142942,0.6921328664,0.03289615512,0.9432430506,0.8282943606,0.6529556513,0.40200001,0.8382412791,0.8785275817,0.08005668471,0.4928152204
+CP-JKU mn40_as (e1+l+all_se),https://github.com/fschmid56/EfficientAT_HEAR,0.5026644483,0.9533687949,0.6399549484,0.8130252361,0.961499989,0.6312075533,0.9079999983,0.9839743614,0.9107142857,0.7253496408,0.0186244165,0.9713343859,0.9046840429,0.6354680061,0.3039999902,0.7766871452,0.8472392559,0.2019387526,0.3179222822
```

Comment:
Please remove the newline at the end of this csv file as it adds a blank row to the table.

Reply:
Dear @jorshi, thanks for reviewing! I've struggled a bit with the GitHub online editor which insists on CSV files ending with a linebreak but it should be done now for both branches.